src/lib.rs

```diff
@@ -945,6 +990,7 @@ pub fn delegate(tokens: TokenStream) -> TokenStream {
                 #(#attrs)*
                 #inline
                 #visibility #signature {
+                    //
```

Comment:
Forgotten line? :)

Reply:
Crap it was my marker for where I could inject the constant getter if you asked for it 🤣😅